tests/conftest.py

```diff
@@ -32,22 +39,20 @@ def session_maker() -> sessionmaker[Session]:
 
 
 @pytest.fixture(scope='session')
-def list_of_db_tables(log: Logger, session_maker: sessionmaker[Session]) -> List[str]:
+def list_of_db_tables(log: logging.Logger, session_maker: sessionmaker[Session]) -> List[str]:
     # log.info('Getting list of db tables')
     session = session_maker()
     try:
-        result = session.execute(
-            text('SELECT tablename FROM pg_tables WHERE schemaname = \'public\'')
-        )
+        result = session.execute(text("SELECT tablename FROM pg_tables WHERE schemaname = 'public'"))
         return [row[0] for row in result if row != 'alembic_version']
     except Exception as e:
         raise e
     finally:
         session.close()
 
 
-@pytest.fixture(scope='function')
-def clear_db_tables(log: Logger, session_maker: sessionmaker[Session], list_of_db_tables: List[str]):
+@pytest.fixture(scope='session')
```

Comment:
@IanHopkinson , can you give more details on the change to the _scope_ of the fixture ? With "session" doesn't it mean that the db will be cleaned only once per testing session ? I was thinking that each test would expect to have a clean db.

Reply:
I tried making this "function" level but the tests seem to take forever to run, I'm trying to diagnose why

Reply:
Thanks for your fix @alexandru-m-g - the tests now pass and the fixtures are refreshed at the function level.

I believe this is ready to merge now.